### PR TITLE
SMPP update SystemID details

### PR DESCRIPTION
**Pull request by mdouchement**

### CODE REVIEW

**Comment by gautierenaud** [comment]
LGTM

File: pages/telecom/sms/smpp-specification/guide.en-gb.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 title: SMPP Technical Specifications
 excerpt: 'Discover the technical specifications of the OVHcloud SMPP solution'
-updated: 2023-07-25
+updated: 2023-07-31
```
Comment (OVH-Team-Guides) [inline pages/telecom/sms/smpp-specification/guide.en-gb.md]:
```suggestion
updated: 2023-08-18
```

File: pages/telecom/sms/smpp-specification/guide.en-gb.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 ---
 
-**Last updated 25th July 2023**
+**Last updated 31st July 2023**
```
Comment (OVH-Team-Guides) [inline pages/telecom/sms/smpp-specification/guide.en-gb.md]: This line can now be removed

File: pages/telecom/sms/smpp-specification/guide.en-gb.md (hunk 1 above)
Comment (mdouchement) [reply pages/telecom/sms/smpp-specification/guide.en-gb.md]: Would be great if a GH action can do this automatically :)

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @mdouchement

**Pull request merged**